cytominer_database/ingest.py

```diff
@@ -103,14 +103,38 @@ def into(input, output, name, identifier, skip_table_prefix=False):
             con = engine.connect()
 
             df = pd.read_csv(source, index_col=0)
-            df.to_sql(name=name, con=con, if_exists="append")
+    """
+        
+    with warnings.catch_warnings():
+        # Suppress the following warning on Python 3:
+        #
+        #   /usr/local/lib/python3.6/site-packages/odo/utils.py:128: DeprecationWarning: inspect.getargspec() is
+        #     deprecated, use inspect.signature() or inspect.getfullargspec()
+        warnings.simplefilter("ignore", category=DeprecationWarning)
+        engine = create_engine(output)
+        con = engine.connect()
+
+        df = pd.read_csv(input)
+        #print("In into(): df['Index']") # no index prepended yet
+        #print(df['Index'])
+        # add "name"prefix to column headers
+        if not skip_table_prefix:
+            no_prefix = ["ImageNumber", "ObjectNumber"]  # exception columns
+            df.columns = [
+                i if i in no_prefix else name + "_" + i for i in df.columns
+            ]
+        # add TableNumber
+        number_of_rows, _ = df.shape
+        table_number_column = [identifier] * number_of_rows  # create additional column
+        df.insert(0, "TableNumber", table_number_column, allow_duplicates=False)    
+        print("In into(): df.shape is ", df.shape) 
+        df.to_sql(name=name, con=con, if_exists="append", index = False)
```

Comment:
```suggestion
        df.to_sql(name=name, con=con, if_exists="append", index=False)
```

We should always be mindful of [pep8](https://www.python.org/dev/peps/pep-0008/) python style guide. In saying this, I also need to confess that _I never think about it when writing my own code_. I only say it here to point out that I observe it when I read other peoples code. 

The reason why I don't think about it is because I use a tool called [black](https://github.com/psf/black). black will automatically format a python file's style and keep things very consistent. I highly recommend!

**BUT** I think we should not use it for this PR. There are already a ton of different things being added here - let's save `black` formatting in a future PR.

Am I making sense here? 😆

Reply:
Thanks for really keeping an eye out here. Of course I read pep8, and many articles related to how to actually stick to it, but as always - easier said than done :) . Will try to be more mindful& run black for newly added files only (as discussed).